server/views/routes/emails.js

```diff
@@ -6,7 +6,7 @@ const responseHandler = require('../../utils/response-handler');
 router.get('/:email', async (req, res) => {
   try {
     const { email } = req.params;
-    const userEmail = await User.getEmail(email);
+    const userEmail = await User.findByEmail(email);
 
     if (userEmail && userEmail.length === 0) {
```

Comment:
Since you're getting back an object here, you want to check if the object exists and that it's not empty. The check for length may fail as it is not an array or string.

Reply:
My bad. Before I was using Document.find() and that was returning an array so I was just checking for its length to see if it was empty. Now that I am using document.findOne() it returns an object. A simple if statement will suffice here 
`if (userEmail)`. If email is not found, null will be returned which is a falsy value and otherwise it will return the actual object which is a truthy